pkg/apis/openebs.io/upgrade/v1alpha1/taskconfig.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+)
+
+// ExecutorConfig represents task executor config using these config
+// task executor runs castemplate engine for each given resources.
+// This contains Versions, upgrade Options and castemplate name.
+type ExecutorConfig struct {
+	Versions
+	Options
+	// Castemplate contain castemplate name which task executor
+	// will use to upgrade single unit of resources.
+	Castemplate string `json:"Castemplate"`
```

Comment:
Can we make this camelCased i.e. `casTemplate`?

Reply:
done